server/routes/testutils/testData.ts

```diff
@@ -1,25 +1,31 @@
-import { Prison } from '../../data/prisonRegisterApiTypes'
+import { PrisonRegisterPrison } from '../../data/prisonRegisterApiTypes'
+import { Prison } from '../../data/visitSchedulerApiTypes'
 
 export default class TestData {
+  // PrisonDto from Visit Scheduler
```

Comment:
I've hacked this around a bit. Got a bit confusing with all the 'prison' variants that seemed handy so I've tried to comment what they are representing. Perhaps a simpler way will emerge as we go along...